.editorconfig

```diff
@@ -15,9 +15,6 @@ indent_size = 4
 
 [*.{kt,kts}]
 ij_kotlin_imports_layout=*
-# Ideally, no experimental rule should be disabled. Ktlint should follow the dogfooding principle. This means that an
-# experimental rule should only be added to the master branch no sooner than that this rule has been applied on the
-# ktlint code base itself.
```

Comment:
Technically `ij_kotlin_allow_trailing_comma` is already a standard rule and does not apply to the comment. This comment was added in https://github.com/pinterest/ktlint/pull/1307.

I will revert my removal here since it is tangential.